nft/src/lib.rs

```diff
@@ -1069,4 +1092,8 @@ impl<T: Config> Pallet<T> {
 			false
 		}
 	}
+
+	fn get_enclave(account: &T::AccountId) -> Option<(ClusterId, EnclaveId)> {
```

Comment:
Does it make sense to have a separate function that just has one line?
You probably should just directly call `T::TEEExt::ensure_enclave` instead of having a wrapper around it.